pkgs/by-name/py/pyfa/package.nix

```
@@ -1,28 +1,114 @@
 {
   lib,
-  appimageTools,
-  fetchurl,
+  python3Packages,
+  fetchFromGitHub,
+  gsettings-desktop-schemas,
+  adwaita-icon-theme,
+  wrapGAppsHook3,
+  gdk-pixbuf,
 }:
 let
-  version = "2.60.2";
+  version = "2.61.0";
 in
-appimageTools.wrapType2 {
+python3Packages.buildPythonApplication rec {
   inherit version;
   pname = "pyfa";
+  format = "other";
 
-  src = fetchurl {
-    name = "pyfa-appimage-${version}";
-    url = "https://github.com/pyfa-org/Pyfa/releases/download/v${version}/pyfa-v${version}-linux.AppImage";
-    hash = "sha256-6doetQ6E1Occ/SqewfxRqPEX1MnUuFomm+8VmetIz4Y=";
+  src = fetchFromGitHub {
+    owner = "pyfa-org";
+    repo = "Pyfa";
+    rev = "v${version}";
```

Prefer prefixing tags with `refs/tags/` to avoid collision with branches
```suggestion
    rev = "refs/tags/v${version}";
```